homework2.py

```diff
@@ -0,0 +1,11 @@
+#Петя, Катя и Сережа делают из бумаги журавликов. 
+#Вместе они сделали S журавликов. Сколько журавликов 
+#сделал каждый ребенок, если известно, что Петя и 
+#Сережа сделали одинаковое количество журавликов, 
+#а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+a = int(input())
+k = int((a/3)*2)
+p = int((k/2)/2)
+s = int(p)
```

Comment:
Не помешала бы проверка входных данных на кратность 6. Кроме того, совершенно необязательно использовать функцию `int`. Достаточно использовать целочисленное деление. В-третьих, приоритет операции, деления и умножения равнозначен.
```suggestion
k = a // 3 * 2
p = s = k // 2 // 2
```